WordPress/Classes/ViewRelated/Blog/QuickStartChecklistViewController.swift

```diff
@@ -157,7 +157,7 @@ private struct QuickStartChecklistConfiguration {
 }
 
 private enum Constants {
-    static let estimatedRowHeight: CGFloat = 90.0
+    static let estimatedRowHeight = CGFloat(90.0)
```

Comment:
I believe the previous form of `let estimatedRowHeight: CGFloat = 90.0` is [preferred](url
https://google.github.io/swift/#initializers-1).